sources/Interop/WinTrust/WinCodec.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+namespace TerraFX.Interop
```

Comment:
Don't think you meant to include this file 😄

Reply:
Fixed.